elastic/apm-server
# Don't drop RUM unsampled transactions

## Motivation/summary

The User Experience app in Kibana currently relies on transaction events only, and not transaction metrics. Store all RUM transaction events so the UX app can calculate statistics correctly.

## Checklist

~- [ ] Update [package changelog.yml](https://github.com/elastic/apm-server/blob/master/apmpackage/apm/changelog.yml) (only if changes to `apmpackage` have been made)~
~- [ ] Documentation has been updated~

## How to test these changes

1. Run apm-server, send unsampled RUM transactions and unsampled backend transactions
2. Check that only the RUM transactions are stored

## Related issues

Closes #6260
